pkg/harvester/edit/harvesterhci.io.storage/provisioners/driver.longhorn.io.vue

```diff
@@ -40,20 +53,30 @@ export default {
     },
   },
 
+  async fetch() {
+    const inStore = this.$store.getters['currentProduct'].inStore;
+
+    await this.$store.dispatch(`${ inStore }/findAll`, { type: NAMESPACE });
+
+    const allSecrets = await this.$store.dispatch(`${ inStore }/findAll`, { type: SECRET });
+
+    // only show non-system secret to user to select
+    this.secrets = allSecrets.filter(secret => secret.isSystem === false);

```

Comment:
@Yu-Jack , @bk201 , as we discussed, show single secret downdrop and only display non-system namespace secret for user to select.

<img width="1492" alt="Screenshot 2024-09-18 at 1 03 22 PM" src="https://github.com/user-attachments/assets/ff84da91-527b-49c4-9bde-49b75360e9f4">